### PR TITLE
feat(pixelknecht, commanderer-backend): add scaling for images

### PR DESCRIPTION
**Pull request by rubenhoenle**

### CODE REVIEW

**Comment by rubenhoenle** [comment]
frontend support will be done within a follow up PR: https://github.com/rubenhoenle/pixelknecht/issues/13

File: pixelknecht/utils.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package main
+
+import (
+	"math"
+)
+
+func CompareFloat(a, b float64) bool {
```
Comment (Golo300) [inline pixelknecht/utils.go:7]: Why is this implementation necessary?

Comment (rubenhoenle) [reply pixelknecht/utils.go:7]: Well, you can't compare floating point values directly. You have to provide some kind of precision.

You will a lot about that when you put "floating point comparison" into your choice of search engine.

Comment (Golo300) [reply pixelknecht/utils.go:7]: Thanks for explanation

**Review by Golo300 — APPROVED** [review]

**Pull request merged**